pybossa/jobs.py

```diff
@@ -1174,7 +1175,7 @@ def check_failed():
             ttl = current_app.config.get('FAILED_JOBS_MAILS')*24*60*60
             sentinel.master.setex(KEY, ttl, 1)
         if int(sentinel.slave.get(KEY)) < FAILED_JOBS_RETRIES:
-            requeue_job(job_id)
+            requeue_job(job_id, sentinel.master)
```

Comment:
Nice catch for the new signature of requeue_job() in the RQ. Does this change fix all the Un-passed unit tests?

Reply:
This fixed tests failed with "TypeError: missing a required argument: 'connection'"